custom-ports/kaldi/portfile.cmake

```diff
@@ -0,0 +1,21 @@
+vcpkg_from_git(
+   OUT_SOURCE_PATH SOURCE_PATH
+   URL https://github.com/kaldi-asr/kaldi.git
+   REF 4a8b7f673275597fef8a15b160124bd0985b59bd # latest as of October 4th
+   HEAD_REF main
+)
+# Set this so it doesn't try and use git to find the version
+# This is what get_version() inside of the kaldi CMakelists.txt calculated
+set(KALDI_VERSION 5.5)
+
+set(VCPKG_CMAKE_CONFIGURE_OPTIONS "-DFETCHCONTENT_FULLY_DISCONNECTED=FALSE")
+
```

Comment:
I encountered a few other warnings on windows, which I think we can fix.

It looks like this is putting cmake files in the installed lib dir, here: `lib/cmake/kaldi`, which is causing some post-build validation warnings.  I don't think these are needed, though?  If that is true, we can just remove them after building:

```
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib/cmake")
```

It also looks like this publishes a bunch of exes.  These of course only have the exe extension on Windows, not mac.  vcpkg expects these to be in a "tools" directory, not the "bin" directory.  This can be fixed with `vcpkg_copy_tools`:

<details>
<summary>Expand to see code to fix this</summary>

```
vcpkg_copy_tools(TOOL_NAMES 
   farcompilestrings
   farcreate
   farequal
   farextract
   farinfo
   farisomorphic
   farprintstrings
   fstarcsort
   fstclosure
   fstcompile
   fstcompose
   fstconcat
   fstconnect
   fstconvert
   fstdeterminize
   fstdifference
   fstdisambiguate
   fstdraw
   fstencode
   fstepsnormalize
   fstequal
   fstequivalent
   fstinfo
   fstintersect
   fstinvert
   fstisomorphic
   fstlinear
   fstloglinearapply
   fstmap
   fstminimize
   fstprint
   fstproject
   fstprune
   fstpush
   fstrandgen
   fstrelabel
   fstreplace
   fstreverse
   fstreweight
   fstrmepsilon
   fstshortestdistance
   fstshortestpath
   fstspecial
   fstsymbols
   fstsynchronize
   fsttopsort
   fstunion
   mpdtcompose
   mpdtexpand
   mpdtinfo
   mpdtreverse
   pdtcompose
   pdtexpand
   pdtinfo
   pdtreplace
   pdtreverse
   pdtshortestpath
   AUTO_CLEAN
)
```

</details>

Reply:
K, I can remove the cmake files since we don't need them.

I can also look into removing the exes.